suite-native/atoms/src/Card/Card.tsx

```diff
@@ -112,10 +115,18 @@ export const Card = React.forwardRef<View, CardProps>(
     ) => {
         const { applyStyle } = useNativeStyles();
 
+        const isAlertDisplayed = !!alertProps;
+        const alertPosition = isAlertDisplayed ? (alertPositionProp ?? 'top') : undefined;
+
         return (
             <View style={applyStyle(cardOuterContainerStyle, { flex: style?.flex })}>
-                {!!alertProps && alertPosition !== 'bottom' && (
-                    <View style={applyStyle(alertBoxWrapperStyle, { alertPosition })}>
+                {isAlertDisplayed && alertPosition === 'top' && (
```

Comment:
Then this could be simplified as 
```suggestion
                {alertPosition === 'top' && (
```